client/me/purchases/purchase-item/index.jsx

```diff
@@ -410,8 +455,10 @@ class PurchaseItem extends Component {
 				} );
 			}
 
-			return translate( 'for {{button}}%(site)s{{/button}}', {
+			// translators: The string contains the name of the site, and the URL of the site e.g. for Block Store (blockstore.com)
+			return translate( 'for {{button}}%(siteName)s{{/button}} ({{link}}%(siteDomain)s{{/link}})', {
 				args: {
+					siteName: site.name,
 					site: site.domain,
```

Comment:
Don't forget to change this to `siteDomain`.

Reply:
👍